exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Function.kt

```diff
@@ -146,7 +146,11 @@ class Min<T : Comparable<T>, in S : T?>(
     /** Returns the expression from which the minimum value is obtained. */
     val expr: Expression<in S>,
     columnType: IColumnType
-) : Function<T?>(columnType) {
+) : Function<T?>(columnType), WithColumnType {
+
+    override val columnType: IColumnType
```

Comment:
Should we mark `Function` with this interface instead?

Reply:
@e5l This causes the same tests to fail as the first commit 😕